lib/cirro_io_v2/resources/base.rb

```diff
@@ -2,7 +2,7 @@
 
 module CirroIOV2
   module Resources
-    class Base
+    class Base < JsonApiClient::Resource
```

Comment:
API v2 is not based on JsonApi anymore. So you can't use JsonApiClient::Resource gem